.github/CODEOWNERS

```diff
@@ -19,9 +20,11 @@
 /packages/rewriter/  @cypress-io/test-runner
 /packages/root/  @cypress-io/test-runner
 /packages/runner/  @cypress-io/test-runner
+/packages/runner-shared/  @cypress-io/test-runner
```

Comment:
Should we also add a runner-ct code owner and respective GH team?

Reply:
@lmiller1990 Yah, was leaving that to the CT team to decide if you want to have your team pinged whenever the code is touched in the packages you 'own'. It's been working well for us because things opened by outside contributors or other teams is pretty obvious if it touches something we should look at without looking through all the files.